src/pages/projectView/projectCreatorViews/sections/rewards/subviews/Shared/ProjectRewardForm.tsx

```diff
@@ -95,6 +95,13 @@ export const ProjectRewardForm = ({
   )
   const [formError, setFormError] = useState<any>({})
 
+  if(!reward.rewardType) {
```

Comment:
The default should be "Select" (ie: `null` value)

Reply:
Mick confirmed to default it to Physical on the reward form.  This is only for the edit/create form.